tasks/services/works.js

```diff
@@ -24,6 +24,7 @@ const hooks = {
     create: [
       iff(taskRecipeIsCompleteOrderSetup, createCastOrderIntentTaskPlan),
       iff(taskRecipeIsCompleteOrderSetup, createCommitOrderTaskPlan),
+      iff(taskRecipeIsStartOrder, sendStartOrderEmails),
```

Comment:
name of `sendStartOrderEmails` is confusing, you also have `sendOrderStartEmail`

Reply:
dealt with this by changing names of the lower order functions~